### ISSUE TITLE
Is 2.1.7RC supported?

### ISSUE THREAD

**Issue by Drakenroft** [technicaljicama/godot-psp#12]
https://downloads.tuxfamily.org/godotengine/2.1.7/rc/20200815/
Since it has just a few number of small fixes to 2.1.6 will it work?

Also an another small question just to be sure about a thing, as far as i read all mesh should be imported as a dae scene and not as mesh right?

Thank you for your amazing work!

**Comment by technicaljicama** [comment]
Yes, this is based on 2.1.7 latest. I didn't manage to compile a working editor with 2.1.7, hence why I linked a 2.1.6 editor build. Yes, dae only. Animations and skeletons work as well.

**Comment by Drakenroft** [comment]
Thanks a lot!
There are any plans to support Godot 3 GLES2 render? Since it has OpenGL2.1/GLES2 same as Godot 2 but is far more usable

Also is there any discord or community forum? I'm having a bit of trouble with DAE and animations even with the blender godot plugin

**Comment by technicaljicama** [comment]
The PSP does not support gles2 and currently I am using a wrapper for gles1. Godot 3 would also be too memory hungry.
No.
I didn't use any blender plugins, I just had a working animation and exported all as a .dae

**Comment by Kystan** [comment]
I apologize in advance for the stupid question, but how do I actually install your, as I understood it, "Disassembled" engine

**Comment by Kystan** [comment]
I apologize in advance for the stupid question, but how do I actually install your, as I understood it, "Disassembled" engine

> The PSP does not support gles2 and currently I am using a wrapper for gles1. Godot 3 would also be too memory hungry. No. I didn't use any blender plugins, I just had a working animation and exported all as a .dae

I apologize in advance for the stupid question, but how do I actually install your, as I understood it, "Disassembled" engine

**Comment by technicaljicama** [comment]
There is no disassembled engine, this is all I have.

**Comment by c08oprkiua** [comment]
> I apologize in advance for the stupid question, but how do I actually install your, as I understood it, "Disassembled" engine
> 
> > The PSP does not support gles2 and currently I am using a wrapper for gles1. Godot 3 would also be too memory hungry. No. I didn't use any blender plugins, I just had a working animation and exported all as a .dae
> 
> I apologize in advance for the stupid question, but how do I actually install your, as I understood it, "Disassembled" engine

To compile it, you need to have the same dependencies installed as what you'd need to compile Godot 2.1 (idk what those are), and then probably PSPSDK (which is an open source PSP toolchain/library suite)
